tests/lax_numpy_reducers_test.py

```diff
@@ -866,5 +866,19 @@ def testCumulativeSumBool(self):
                              dtype=jnp.bool_)
     np.testing.assert_array_equal(np.array([[True], [True], [False]]), out)
 
+  @jtu.sample_product(dtype=[dtypes.int4, dtypes.uint4, dtypes.int2, dtypes.uint2])
+  @jtu.ignore_warning(category=NumpyComplexWarning)
+  def test_low_bit_sum(self, dtype) -> None:
+    if dtype is not None:
+      rng = jtu.rand_default(self.rng())
+      shape = (10, 10)
+      tol = 0.01
+      args_maker = lambda: [rng(shape, dtype)]
+      sum_jit = jax.jit(jnp.sum)
+      np_dtype = np.uint32 if dtypes.issubdtype(dtype, np.unsignedinteger) else np.int32
+      np_sum = lambda x: np.sum(x.astype(np_dtype))
+      self._CheckAgainstNumpy(sum_jit, np_sum, args_maker, tol=tol)
```

Comment:
Let's use `tol=0`, because we expect an exact match for integer sums.

Reply:
indeed, it makes sense :)